Jenkinsfile_CNP

```diff
@@ -7,7 +7,7 @@ def type = "nodejs"
 def product = "civil"
 def component = "ccd"
 def camundaBranch = "master"
-def dmnBranch = "master"
+def dmnBranch = "feat/civ-0000-query-management"
```

Comment:
Before merging we will need to remember this DMN branch or create spearate test PR.